web-test-runner.config.js

```diff
@@ -0,0 +1,40 @@
+const path = require("path");
+const { fromRollup } = require("@web/dev-server-rollup");
+const { esbuildPlugin } = require("@web/dev-server-esbuild");
+const { importMapsPlugin } = require("@web/dev-server-import-maps");
+const rollupAlias = require("@rollup/plugin-alias");
+
+const alias = fromRollup(rollupAlias);
+
+const projectRootDir = path.resolve(__dirname);
+
+const MOCKS = "/frontend/tests/mocks";
+
+module.exports = {
+  nodeResolve: true,
+  plugins: [
+    esbuildPlugin({ ts: true }),
+    importMapsPlugin({
+      inject: {
+        importMap: {
+          imports: {
+            // use mobx dev build to avoid error due to `process.env`
+            mobx: "./node_modules/mobx/dist/mobx.esm.development.js",
```

Comment:
By default, `mobx` checks for development mode with `process.env.NODE_ENV`

Even if we use `dist/mobx.esm.js` those checks are still there. So I had to use this.